tsconfig.strict.json

```diff
@@ -247,6 +247,13 @@
     "lib/workers/repository/onboarding/pr/errors-warnings.ts",
     "lib/workers/repository/onboarding/pr/index.ts",
     "lib/workers/repository/onboarding/pr/pr-list.ts",
+    "lib/workers/repository/config-migration/branch/commit-message.ts",
```

Comment:
do we need all files excluded because of transitive dependencies?

Reply:
ive now handled the ones that i can in this PR scope,
left only the transitive deps.